Rakefile

```diff
@@ -1,8 +1,9 @@
-require 'bundler/setup'
```

Comment:
Not forcing bundler just leads to strange version conflicts ... so I prefer
forcing it to not get invalid bug reports or people running into strange
issues.

Afaik there is no upside of adding anything to dev dependencies except for
documentation ... but not reason not to.

On Sun, Jul 31, 2016 at 6:20 AM, Jason Karns <notifications@github.com>
wrote:

> In Rakefile
> <https://github.com/gregorym/bump/pull/59#discussion_r72904055>:
>
> > @@ -1,8 +1,9 @@
> > -require 'bundler/setup'
>
> With bundler/setup included in the rakefile, this *forces* bundler to be
> used by all developers. But it needn't be.
>
> Alternatively, if bundler is going to be a required development
> dependency, it needs to be added to the gemspec.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gregorym/bump/pull/59/files/1ba6c60f4a7458a4ad1a719978315fc2d5b0fe98#r72904055>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ6BplTGdFPvgmjT3BOolwwXj-FBeks5qbKEBgaJpZM4JY3CX>
> .
>